Modules/System/Regex/README.md

```diff
@@ -0,0 +1,8 @@
+This module provides an interface, that lets you use regular expresssions.
```

Comment:
The RegEx module is unrelated to your PR. Please revert. The module will get added with the 17.2 updates which I'll push later today.